spamhandling.py

```diff
@@ -103,8 +101,7 @@ def handle_spam(title, body, poster, site, post_url, poster_url, post_id, reason
             username = poster.strip()
             user_link = poster_url
 
-        t_metasmoke = Thread(target=Metasmoke.send_stats_on_post,
```

Comment:
I'm a little confused here? Do you want me to A) break consistency and return it to how it was before or B) leave it as is?